assets/templates/dataset-filter/hierarchy.tmpl

```diff
@@ -96,7 +96,7 @@
                                             </div>
                                             {{ if ne .SubNum "0" }}
                                                 <div class="view-link width-md--10 float-el--right-md text-right--md padding-top--1 height-sm--5 height-md--5">
-                                                    <input name="{{ .SubURL }}" id="{{.ID}}-children" type="submit" class="padding-top--0 text-align-top btn btn--link underline-link" value="Browse {{.SubNum}} {{$.Type}}{{ if ne .SubNum "1" }}s{{end}}"/>
+                                                    <input name="{{ .SubURL }}" id="{{.ID}}-children" type="submit" class="padding-top--0 text-align-top btn btn--link underline-link" aria-label="{{ .Label }}: browse {{.SubNum}} {{$.Type}}" value="Browse {{.SubNum}} {{$.Type}}{{ if ne .SubNum "1" }}s{{end}}"/>
```

Comment:
The conditional for the s does not appear to work. For example I see: `01 Food and non-alcoholic beverages: browse 2 type`

Reply:
Ah, that's for the actual text displayed on screen, rather than what the screen reader announces. If you have a look at the text on screen, it should work. Templates would throw an error if it was `1` instead of `"1"` and fail to render. That said, it probably would be good to include that in the screen reader announcement too, so I can add that in